src/traces/bar/defaults.js

```diff
@@ -70,10 +70,13 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
 
     handleStyleDefaults(traceIn, traceOut, coerce, defaultColor, layout);
 
+    var lineColor = (traceOut.marker.line || {}).color;
+    var markerColor = (traceOut.marker || {}).color;
+
     // override defaultColor for error bars with defaultLine
     var errorBarsSupplyDefaults = Registry.getComponentMethod('errorbars', 'supplyDefaults');
-    errorBarsSupplyDefaults(traceIn, traceOut, Color.defaultLine, {axis: 'y'});
-    errorBarsSupplyDefaults(traceIn, traceOut, Color.defaultLine, {axis: 'x', inherit: 'y'});
+    errorBarsSupplyDefaults(traceIn, traceOut, lineColor || markerColor || Color.defaultLine, {axis: 'y'});
+    errorBarsSupplyDefaults(traceIn, traceOut, lineColor || markerColor || Color.defaultLine, {axis: 'x', inherit: 'y'});
```

Comment:
I don’t think `markerColor` belongs here. That would be invisible against the bars themselves, right? I like `lineColor` though.

Reply:
Yes totally agree.

Reply:
@alexcjohnson there is also [this PR](https://github.com/plotly/plotly.js/pull/3414) for possibly only fixing this on scatter traces...